src/main/java/core/basesyntax/FileReaderImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) throws IOException {
```

Comment:
The method `read` should not throw a checked exception like `IOException`. According to the checklist, you should catch the `IOException` and rethrow it as a `RuntimeException` with an informative message.